src/stylized/BulletListStyle.js

```diff
@@ -1,10 +1,10 @@
 //@flow
 import * as React from 'react'
-import Wrap from '../Wrap'
+import Wrap, { defaultProps } from '../Wrap'
 import type { WrapProps } from '../Wrap'
 
 const BulletListStyle = (props: WrapProps): React.Element<*> => (
-  <Wrap {...props}>
+  <Wrap {...defaultProps} {...props} >
```

Comment:
Aren't the spread of `defaultProps` unnecessary in the stylized components, since this happens in the `Wrap` component as well.

Reply:
actually `Wrap` does not handle `defaultProps` - it only exports them

Reply:
Oh, you're right. I was confused looking at `ContentLoader`. But, I don't se why we couldn't add logic there to do the merging of props in one spot?

Reply:
I had to read `mergedProps` [here](https://github.com/danilowoz/react-content-loader/pull/72/files#diff-1fdf421c05c1140f6d71444ea2b27638R28). I'd love to kill this merging all over the place too - just sent this PR really quickly. If you have any proposal on how I can refactor this, please dont hesitate to speak up.

We can ofc inline those values in the mentioned place and use `props.width || defaultWidth` etc